### PR TITLE
[py] deprecate all but `Options` and `Service` arguments in driver instantiation.

### LINKED ISSUE #9125: [py] deprecate all but `Options` and `Service` arguments in driver instantiation.
## 🐛 Bug Report

We need the Chrome/Chromium/Ie `webdriver.py` class initialisation to only take `Service` and `Options` classes.

To get there we need to deprecate the rest of the arguments

Note: Remote doesn't need this deprecation, at least just yet.

To run the tests you need to `bazel test //py:test-chrome`

## To Reproduce

Just read the code

Detailed steps to reproduce the behavior:

## Expected behavior

A cleaner `__init__` is expected.

## Environment

OS: All
Browser: All
Browser version: 
Browser Driver version: all
Language Bindings version: python
Selenium Grid version (if applicable):

### PR DESCRIPTION
**Pull request by itsjwala**
**Thanks for contributing to Selenium!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://github.com/SeleniumHQ/selenium/blob/trunk/CONTRIBUTING.md) guidelines.
Avoid large PRs, help reviewers by making them as simple and short as possible.

### Description

Deprecated remaining `__init__` params from `chromium` and `IE` webdriver class
I haven't touched the `chrome` driver since all the params are already deprecated if not deprecated in the parent `chromium` class. 
Left out `browser_name` & `vendor_prefix` for chromium driver family

### Motivation and Context

Fixes #9125 

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [x] I have read the [contributing](https://github.com/SeleniumHQ/selenium/blob/trunk/CONTRIBUTING.md) document.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: py/selenium/webdriver/chromium/webdriver.py [hunk 1]
```diff
@@ -35,22 +36,28 @@ class ChromiumDriver(RemoteWebDriver):
...
-                 service: Service = None, keep_alive=True):
+                 service: Service = None, keep_alive=DEFAULT_KEEP_ALIVE):
         """
         Creates a new WebDriver instance of the ChromiumDriver.
         Starts the service and then creates new WebDriver instance of ChromiumDriver.
 
         :Args:
-         - browser_name - Browser name used when matching capabilities.
-         - vendor_prefix - Company prefix to apply to vendor-specific WebDriver extension commands.
+         - browser_name - Deprecated: Browser name used when matching capabilities.
```
Comment (AutomatedTester) [inline py/selenium/webdriver/chromium/webdriver.py]: Actually thinking about this we should keep `browser_name` and `vendor_prefix` due to the way inheritance is done.

**Review by AutomatedTester — COMMENTED** [review]
I think we can clean up `chrome/webdriver.py` here

File: py/selenium/webdriver/chromium/webdriver.py (hunk 1 above)
Comment (itsjwala) [reply py/selenium/webdriver/chromium/webdriver.py]: Removed `browser_name` and `vendor_prefix` from deprecation for chromium family drivers for now.

**Comment by itsjwala** [comment]
> I think we can clean up `chrome/webdriver.py` here

should I deprecate `keep_alive`  from `Opera` and `Edge` driver as well as they are part of `chromium` webdriver?

**Review by AutomatedTester — APPROVED** [review]

**Comment by AutomatedTester** [comment]
Thank you for your contribution!

**Pull request merged**